### PR TITLE
vadc: alpha

### PR DESCRIPTION
**Pull request by zebra6**

### CODE REVIEW

File: zebra_claw/run.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        i.fobj = open(i.name, "w")
+        i.fobj.close()
+
+    log.log(log.ll.info, "starting execution...")
+    run(vlcs)
+
+    #clean up
+    print("deleting virtual load cells")
+
+    for i in data_endpoints:
```
Comment (beardedtrogon) [inline zebra_claw/run.py:50]: I don't see data_endpoints anywhere else. What exactly is this?

File: zebra_claw/run.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    os.rmdir(dirname)
+
+
+###############################################################################
+# actually start doing something
+###############################################################################
+def run(vlc_list):
+    done = False
+
+    while (done != True):
```
Comment (beardedtrogon) [inline zebra_claw/run.py:64]: When is done changed to True?

**Review by beardedtrogon — COMMENTED** [review]
Looks fine. Just take a look at my comments to see if there is anything that was missed

File: zebra_claw/run.py (hunk 2 above)
Comment (zebra6) [reply zebra_claw/run.py:64]: It isn't yet

File: zebra_claw/run.py (hunk 1 above)
Comment (zebra6) [reply zebra_claw/run.py:50]: Letfover from an earlier approach.

**Pull request merged**